includes/js/myPluginsTools.js

```diff
@@ -1,151 +1,177 @@
+	}
 
-    getElementByStatus( isActive, isInstalled, pluginData ) {
-      if( isActive && isInstalled ){ // active and installed
-        return `<a 
-          title="Manage"
-          href="${this.renderCTAUrl(pluginData?.cta?.url)}"
-          class="nfd-solutions-availble-list-item-button"
-        >${pluginData?.cta?.text}</a>`;
-      } else if ( isInstalled ){ // already installed
-        return `<button
-          title="Activate Plugin"
-          data-nfd-installer-plugin-activate="${true}"
-          data-nfd-installer-pls-slug="${pluginData?.plsSlug}"
-          data-nfd-installer-pls-provider="${pluginData?.plsProviderName}"
-          data-nfd-installer-plugin-name="${pluginData?.name}"
-          data-nfd-installer-plugin-url="${this.renderCTAUrl(pluginData?.cta?.url)}"
-          href="${this.renderCTAUrl(pluginData?.cta?.url)}"
-          class="nfd-solutions-availble-list-item-button nfd-activate-btn"
-        >${pluginData?.cta?.text}</button>`;
-      }
+	getElementByStatus( isActive, isInstalled, pluginData ) {
+		// active and installed - no installer attributes needed
+		if ( isActive && isInstalled ) {
+			return `<a 
+          title="Manage Plugin"
+          class="nfd-solutions-availble-list-item-button nfd-solutions-${ this.renderNameAsClass(
+				pluginData?.name
+			) }-button"
+          href="${ this.renderCTAUrl( pluginData?.cta?.url ) }"
+        >${ pluginData?.cta?.text }</a>`;
+		}
 
-      if ( pluginData?.plsProviderName && pluginData?.plsSlug ){ // premium plugin with pls
-        return `<button
-          title="Install Premium Plugin"
-          class="nfd-solutions-availble-list-item-button"
-          data-nfd-installer-plugin-activate="${true}"
-          data-nfd-installer-pls-slug="${pluginData?.plsSlug}"
-          data-nfd-installer-pls-provider="${pluginData?.plsProviderName}"
-          data-nfd-installer-plugin-name="${pluginData?.name}"
-          data-nfd-installer-plugin-url="${this.renderCTAUrl(pluginData?.cta?.url)}"
-        >${pluginData?.cta?.text}</button>`;
-      } else if ( pluginData?.download ) { // free plugin
-        return `<button
-          title="Install Plugin"
-          class="nfd-solutions-availble-list-item-button"
-          data-nfd-installer-pls-provider="${pluginData?.plsProviderName}"
-          data-nfd-installer-plugin-activate="${true}"
-          data-nfd-installer-plugin-name="${pluginData?.name}"
-          data-nfd-installer-download-url="${pluginData?.download}"
-          data-nfd-installer-plugin-url="${this.renderCTAUrl(pluginData?.cta?.url)}"
-        >${pluginData?.cta?.text}</button>`;
-      } else { // fallback
-        return `<a
+		// premium plugin - has pls values
+		if ( pluginData?.plsProviderName && pluginData?.plsSlug ) {
+			return `<button
+          title="${ isInstalled ? 'Activate' : 'Install' } Premium Plugin"
+          class="nfd-solutions-availble-list-item-button nfd-solutions-${ this.renderNameAsClass(
```

Comment:
I added this little method `renderNameAsClass` to give us unique class names for each plugin (which helps select them in tests). I also update the order of these installer attributes so they are consistent in each block. The rest of the file changes are due to lint whitespace edits.